Visualize part review edits (#1348)
hadley/r4ds: EDA.qmd

      filter(between(y, 3, 20))
    ```

>     We don't recommend this option because, for a given observation, just because the value for one variable is invalid doesn't mean the values for all variables are.

Reviewer: ```suggestion
    We don't recommend this option because one invalid value doesn't imply that all the other values for that observation are also invalid.
```

      mutate(y = if_else(y < 3 | y > 20, NA, y))
    ```

> `if_else()` has three arguments.

Reviewer: IMO this is the biggest benefit of the TRs — spotting when we're moved stuff so we accidentally use it before we teach it.

Author: Agreed, those have been super helpful!

  geom_freqpoly(aes(color = cut), binwidth = 500, linewidth = 0.75)
```

Note that ggplot2 uses an ordered color scale for `cut` because it's defined as an ordered factor variable in the data.
You'll learn more about these in @sec-ordered-factors.

> The default appearance of `geom_freqpoly()` is not that useful for that sort of comparison because the height is given by the count and the overall counts of `cut` in differ so much, making it hard to see the differences in the shapes of their distributions.

Reviewer: ```suggestion
The default appearance of `geom_freqpoly()` is not that useful here because the height, determined by the overall count, differs so much across `cut`s, making it hard to see the differences in the shapes of their distributions.
```

(In this section we'll use the `smaller` dataset to stay focused on the bulk of the diamonds that are smaller than 3 carats)

> Scatterplots become less useful as the size of your dataset grows, because points begin to overplot, and pile up into areas of uniform black, making it hard to see the trend.

Reviewer: ```suggestion
Scatterplots become less useful as the size of your dataset grows, because points begin to overplot, and pile up into areas of uniform black, making it hard to see where distribution in space.
```

That's maybe not quite right either, but it's definitely the 2d distribution that it affects, not just the ability to judge trend.

Author: Agreed, replacing with 

> making it hard to judge differences in the density of the data across the 2-dimonsional space as well as making it hard to spot the trend.
